assignment-1/src/index.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+const HelloWorld = () => {
```

Comment:
Can we create a separate folder named `components` and place the `HelloWorld`, `Text`, `InputText` components there like `src/components/HelloWorld/index.js`

also for `src/index.js` lets create `container/HomePage/index.js` and put the code there and import that container in `src/index.js`